docs/api/README.md

```diff
@@ -1080,7 +1081,7 @@ A channel is an object used to send and receive messages between tasks. Messages
 
 Every channel has an underlying buffer which defines the buffering strategy (fixed size, dropping, sliding)
 
-The Channel interface defines 3 methods: `take`, `put` and `close`
+The Channel interface defines 4 methods: `take`, `put`, `flush` and `close`
```

Comment:
Actually flush is not available on multicastChannel and put is not available on eventChannel.

Reply:
@Andarist well, I think this description could be added below this section, on those methods rules